spec/lib/mastodon/cli/accounts_spec.rb

```diff
@@ -345,4 +345,49 @@
       end
     end
   end
+
+  describe '#follow' do
+    context 'when the given username is not found' do
+      let(:arguments) { ['non_existent_username'] }
+
+      it 'exits with an error message indicating that no account with the given username was found' do
+        expect { cli.invoke(:follow, arguments) }.to output(
+          a_string_including('No such account')
+        ).to_stdout
+          .and raise_error(SystemExit)
+      end
+    end
+
+    context 'when the given username is found' do
+      let!(:target_account) { Fabricate(:account) }
+      let!(:local_account1) { Fabricate(:account) }
```

Comment:
There is a new rule in Rubocop being enabled in https://github.com/mastodon/mastodon/pull/24698 that flags these numbered variables. May be worth thinking about a better name now before that lands or this lands and it needs to be updated in that PR

Reply:
Thank you for pointing that. I'll change it.